service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/AbstractEmailContentProvider.java

```diff
@@ -67,7 +67,8 @@ protected AbstractEmailContentProvider(String uiBaseUrl, DateFormatterService da
     private String getHearingBooking(CaseData data) {
```

Comment:
I know it's not your code, but method called hearingBooking when we return date is far from ideal.